ui/raidboss/data/03-hw/trial/sephirot-ex.ts

```diff
@@ -1,13 +1,322 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  mainTank?: string;
+  phase: number;
+  force?: string;
+  shakerTargets?: string[];
+  pillarActive: boolean;
+}
 
 const triggerSet: TriggerSet<Data> = {
   zoneId: ZoneId.ContainmentBayS1T7Extreme,
   timelineFile: 'sephirot-ex.txt',
-  triggers: [],
+  initData: () => {
+    return {
+      phase: 1,
+      pillarActive: false,
+    };
+  },
+  timelineTriggers: [
+    {
+      id: 'SephirotEx Tiferet',
+      regex: /Tiferet/,
+      beforeSeconds: 4,
+      suppressSeconds: 2, // Timeline syncs can otherwise make this extra-noisy
```

Comment:
Yeah, I've been wondering if we just need some built-in logic to auto-suppress timeline triggers.